admin_organization_test.go

```diff
@@ -0,0 +1,218 @@
+package tfe
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAdminOrganizations_List(t *testing.T) {
+	skipIfCloud(t)
+
+	client := testClient(t)
+	ctx := context.Background()
+
+	t.Run("with no list options", func(t *testing.T) {
+		org, orgTestCleanup := createOrganization(t, client)
+		defer orgTestCleanup()
```

Comment:
Moved this one out because it was used twice, but the others I kept because I only ever used `createOrganization()` in one part of the test and wasn't used in multiple `t.Run()`s.